src/utils.js

```diff
@@ -27,3 +27,11 @@ export let findKeys = _.curry((predicate, data) =>
 export let coalesce = _.find(F.isNotNil)
 
 export let optionsFromArray = F.mapIndexed((label, value) => ({ label, value }))
+
+export let getVariants = (props, variants, defaultKey) =>
```

Comment:
This could use a unit test or at least a comment documenting its usage

Reply:
commented - I'd like to add a unit test at some point but now is probably not the best time